services/PostService.js

```diff
@@ -0,0 +1,74 @@
+const postModel = require('../models/Post');
+const BaseService = require('./BaseService');
+
+module.exports = class extends BaseService {
+
+  constructor() {
+    super();
+  }
+
+  async index() {
+
+    try {
+      const posts = await postModel.find().skip(0).limit(5); // todo: make pagination
```

Comment:
For pagination you should create paginate method.
`
paginate(page, count){ ... }
`